components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import MaterialButton from '@material-ui/core/Button';
+import { styled } from '@material-ui/core/styles';
+
+const StyledButton = styled(MaterialButton)({
+  color: 'red',
+});
```

Comment:
We use styled components from material-ui to apply any styling overrides to the components, these are styles that should be global for the app, but cannot be applied using the theme for one reason or another.